create-directories.bash

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion
set -o errexit
set -o nounset
set -o pipefail
```

Reply:
Added this and a `trap` to provide a better error message.